File: TaskWebApp/Model/TodoTask.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace TaskWebApp.Model
{
    public class TodoTask
    {
        public int taskid { get; set; }
        public string TaskName { get; set; }
        public string CreatorName { get; set; }
        public string Description { get; set; }
        public DateTime CreationDate { get; set; }
        public bool IsFinished { get; set; }

    }
}

File: TaskWebApp/ClientApp/src/app/model/TodoTask.ts
export class TodoTask {
  public taskid: number;
  public taskName: string;
  public creatorName: string;
  public description: string;
  public creationDate: Date;
  public isFinished: boolean;

  public clear() {
    this.taskid = null;
    this.taskName = this.creatorName = this.description = null;
    this.creationDate = null;
    this.isFinished = false;
  }
}
